presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -100,14 +100,8 @@
     private boolean rcfileOptimizedWriterEnabled;
 
     private HiveMetastoreAuthenticationType hiveMetastoreAuthenticationType = HiveMetastoreAuthenticationType.NONE;
-    private String hiveMetastoreServicePrincipal;
-    private String hiveMetastoreClientPrincipal;
-    private String hiveMetastoreClientKeytab;
-
     private HdfsAuthenticationType hdfsAuthenticationType = HdfsAuthenticationType.NONE;
-    private boolean hdfsImpersonationEnabled;
-    private String hdfsPrestoPrincipal;
-    private String hdfsPrestoKeytab;
+    private boolean hdfsImpersonationEnabled = false;
```

Comment:
Nevermind, I didn't notice the deletion above. This change is actually unnecessary, since it sets the field to the default Java value, which we don't do in our code style (IntelliJ has an inspection for this).